mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java

```diff
@@ -51,6 +52,24 @@ public ChangeManagerImpl(Vertx vertx, String tenantId) { //NOSONAR
     this.tenantId = TenantTool.calculateTenantId(tenantId);
   }
 
+
+  @Override
+  public void deleteChangeManagerJobExecutions(DeleteJobExecutionsDto entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(c -> {
+      try {
+        jobExecutionService.deleteJobExecutionByIds(entity.getIds(), tenantId)
+          .map(optionalJobExecution -> !optionalJobExecution.booleanValue()?
+            new BadRequestException(format("JobExecution with id '%s' was not found", entity.getIds())): optionalJobExecution)
+          .map(Response.class::cast)
+          .otherwise(Response.ok().build())
```

Comment:
Is it possible to rewrite, because otherwise() more suitable for failure cases, not success ones

Reply:
@SerhiiNosko - PR updated as per comments, Thanks 🙂